### DISCUSSION TITLE
{"error": "Rate limited by proxy server"}

### DISCUSSION THREAD

**Discussion by MOHOAzure** [acheong08/ChatGPT#1139] [Q&A]
A few hours ago, I encountered the `revChatGPT.V1.Error: OpenAI: {"error": "Rate limited by proxy server"} (code: 429)` issue when using V1. I thought it might have been due to my requests or the V1 server reaching the OpenAI limit, so I took a break for hours and resumed just now. However, I still encountered this issue. What should I do next to handle this?

**Comment by Worrah** [comment]
same for me, still not resolved in 4.0.4
UPD: after deleting the conversation id all works fine

**Comment by acheong08** [comment]
Your IP address might be shared. As the error implies, I rate limit the bypass server because I currently can't afford high traffic

**Reply by acheong08** [reply]
It is based on IP address at 5 requests per 10 second

**Reply by Worrah** [reply]
but why this error disappears after i've changed the conversation_id then?

**Reply by acheong08** [reply]
Putting in conversation id without parent id makes it try to fetch conversation history. That could send more requests than the limit

**Reply by Worrah** [reply]
umm. where i can find a parent id? ese060f1-c1eb-490d-8a5c-cba0ccddb067 or id's like that already include it?

**Comment by MOHOAzure** [comment]
Oh the `conversion_id`. I just remembered that this error occurred after I put the conversion_id in the ask() function without parent_id.

BTW, my account got banned few days ago. Just like this issue: https://github.com/acheong08/ChatGPT/issues/1158

I thought I got the reason.
